packages/core/state/interaction/actions.ts

```diff
@@ -679,3 +679,47 @@ export function setSelectedPublicDataset(dataset: PublicDataset): SetSelectedPub
         type: SET_SELECTED_PUBLIC_DATASET,
     };
 }
+
+/**
+ * SHOW_MOVE_FILE_MANIFEST
+ *
+ * Action to show the Move File dialog (manifest) for NAS cache operations.
+ * This modal will allow users to move files on or off the NAS cache.
```

Comment:
just on

Reply:
or rather, 'onto'

Reply:
resolved in [1bc859f](https://github.com/AllenInstitute/biofile-finder/pull/311/commits/1bc859fc0ecc021de5b18b02a10e656a50a92322)